frontend/src/entities/configuration/dynamic-configuration.tsx

```diff
@@ -136,15 +216,225 @@ export function DynamicConfiguration () {
                     {<pre>TODO</pre>}
                 </ExpandableSection>
                 <ExpandableSection headerText='EMR Config' variant='default' defaultExpanded>
-                    {<pre>TODO</pre>}
-                    <Input
-                        data-cy='test-input'
-                        value={state.form.configuration.EMRConfig.clusterSizes[0].name}
-                        onChange={(event) => {
-                            setFields({ 'configuration.EMRConfig.clusterSizes[0].name': event.detail.value });
-                        }}
-                        onBlur={() => touchFields(['configuration.EMRConfig.clusterSizes[0].name'])}
-                    />
+                    <Container
```

Comment:
Looking at the screenshots I think we're overusing containers quite a bit here and we're also missing some whitespace in between things. We really shouldn't be nesting containers when we can help it and the current layout of the config page is quite confusing from a UX perspective (I know some of that was done previously).

Reply:
I'll start by removing the containers - should I just make more use of the `ExpandableSection` component? Maybe even have those sections collapsed by default so it doesn't take up so much of the screen.